search-api.md

```diff
@@ -88,6 +90,15 @@ After receiving a request, the remote server can respond in one of two ways:
 }
 ```
 
+#### Disclaimer
+* *Optional*
+* Disclaimer text for the service, this is optional and can be included in all requests and responses. If included it is assumed that it supersedes the disclaimer stored in the GitHub repository.
+* Note that the diclaimer should be shown with the returned data per item 6 of the MME service agreement.
```

Comment:
GeneMatcher includes the disclaimer and guidelines for citation in the email we return to our submitters after they make an MME request. This information is currently stored in a table in our database but will likely be updated from the disclaimer included in the responses in the future. I know *we* have agreed to the service agreement but we need to make sure users get that too.